docs/manual/faq/_index.de.md

```diff
@@ -166,3 +166,14 @@ die Datei `task.json`.
 Anschließend sollte der Contao Manager wieder laufen.
 {{% /expand %}}
 
+{{% expand "Kann ich die .phar Datei umbenennen?" %}}
```

Comment:
{{% expand "Kann ich die `.phar` Datei umbenennen?" %}}

Reply:
I am not sure you can use back ticks in there.

Reply:
What about ... ».phar« ...?

Reply:
Done